src/components/JsonInput.vue

```diff
@@ -17,8 +17,10 @@
   import { stringify } from '@/utilities/json'
 
   const props = defineProps<{
+    minLines?: number,
```

Comment:
I added these purely for the schema form `withPropsWithoutExcludedFactory` type, which only pulls props that are exposed from the component, not any that it wraps; there might be a better way to handle this

Reply:
this makes sense to me. This is a general problem that soon we'll be able to fix (in vue 3.3) by making props their own types and exporting them from the component itself (cannot export multiple from a sfc today). Then in a component like this you could import the type and add it to your props (cannot use an imported type for defineProps today). Both fixed in 3.3